### PR TITLE
feat(docs): `.replace()` and `.replaceGet()`

### PR DESCRIPTION
**Pull request by novusnota**
## Issue

Docs towards #538 and alongside PR #941.

## Checklist

- [ ] I have updated CHANGELOG.md. Didn't do that yet, since the [feature PR](https://github.com/tact-lang/tact/pull/941) isn't merged yet.
- [x] I have run the linter, formatter and spellchecker
- [x] I did not do unrelated and/or undiscussed refactorings

### CODE REVIEW

**Comment by anton-trunov** [comment]
Let's move the contents of this PR to #941 as it should be

File: docs/src/content/docs/book/maps.mdx [hunk 1]
```diff
@@ -96,6 +96,90 @@ if (gotButUnsure != null) {
...
+```tact
+// Empty map
+let fizz: map<Int, Int> = emptyMap();
+
+// Setting a couple of values under different keys
+fizz.set(7, 70);
+fizz.set(42, 42);
+
+// Overriding one of the existing key-value pairs
+let oldVal1 = fizz.replace(7, 68); // key 7 now points to value 68
```
Comment (anton-trunov) [inline docs/src/content/docs/book/maps.mdx]:
```suggestion
let oldVal1 = fizz.replaceGet(7, 68); // key 7 now points to value 68
```

File: docs/src/content/docs/book/maps.mdx [hunk 2]
```diff
@@ -96,6 +96,90 @@ if (gotButUnsure != null) {
...
+oldVal1; // 70
+
+// Trying to replace the value in a non-existing key-value pair will do nothing
+let oldVal2 = fizz.replace(8, 68); // no key 8, so nothing was altered
```
Comment (anton-trunov) [inline docs/src/content/docs/book/maps.mdx]:
```suggestion
let oldVal2 = fizz.replaceGet(8, 68); // no key 8, so nothing was altered
```

File: docs/src/content/docs/book/maps.mdx [hunk 3]
```diff
@@ -96,6 +96,90 @@ if (gotButUnsure != null) {
...
+```tact
+// Empty map
+let fizz: map<Int, Int> = emptyMap();
+
+// Setting a couple of values under different keys
+fizz.set(7, 70);
+fizz.set(42, 42);
+
+// Overriding one of the existing key-value pairs
+let oldVal1 = fizz.replace(7, null); // the entry under key 7 is now deleted
```
Comment (anton-trunov) [inline docs/src/content/docs/book/maps.mdx]:
```suggestion
let oldVal1 = fizz.replaceGet(7, null); // the entry under key 7 is now deleted
```

File: docs/src/content/docs/book/maps.mdx [hunk 4]
```diff
@@ -96,6 +96,90 @@ if (gotButUnsure != null) {
...
+oldVal1; // 70
+
+// Trying to replace the value in a non-existing key-value pair will do nothing
+let oldVal2 = fizz.replace(8, null); // no key 8, so nothing was altered
```
Comment (anton-trunov) [inline docs/src/content/docs/book/maps.mdx]:
```suggestion
let oldVal2 = fizz.replaceGet(8, null); // no key 8, so nothing was altered
```

**Review by anton-trunov — CHANGES_REQUESTED** [review]

**Comment by novusnota** [comment]
Moved the contents of this PR to #941 in this commit: https://github.com/tact-lang/tact/pull/941/commits/867e5fe975d4fd0f3f31418718b825b57523c8d5. Thus, this PR shall be closed :)

**Pull request closed without merging**